### ISSUE TITLE
Ensure that the user can use a custom font size

### ISSUE THREAD

**Issue by rreitmann** [dzhw/metadatamanagement#1062]
In order to improve accessibility we must ensure that the user can use a custom (bigger) font size. Currently the text is not readable in some places if increase the min font size (search card header). This feature must nor interfere with zooming, which currently works as expected.
